modules-sdks/pax-terminal/instore-solution/NET/Methods/refundasync.md

```diff
@@ -0,0 +1,25 @@
+---
+title: RefundAsync
+description: async Task\<PaymentRequestResult\> RefundAsync(decimal amount,string apmreference="", [string currency="SEK"])
+---
+
+The RefundAsync should be called when the amount is known. It opens all available readers and waits for a payment instrument. If Alternative Payment Methods are activated it will open for that too.
+
+{% include alert.html type="warning" icon="warning" header="Heads up"
+body="After RefundAsync returns there has to be a delay before next request can be made. If there is no delay the next request will fail, indicating that the terminal is busy and retries have to be made."
+%}
+
+### Parameters
+
+* amount - amount to for refund
+* apmreference - A reference if the transaction was made using APM. When this reference is not null, APM is assumed.
```

Comment:
Good, can be used both for APM and omni reversal (bought online and return in physical store), might want to call it paymentCode since that's what it's called in eCom, or PaymentReceiptId called in APM json receipt